src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -1775,6 +1775,11 @@ int MachCallNativeNode::ret_addr_offset() {
   return -1;
 }
 
+int MachCallBlackholeNode::ret_addr_offset() {
```

Comment:
It can be moved into shared code (`machnode.cpp`).

Reply:
True! Fixed.